src/main/java/com/alrex/parcool/common/compat/shoulderSurfing/ShoulderSurfingCompat.java

```diff
@@ -0,0 +1,58 @@
+package com.alrex.parcool.common.compat.shoulderSurfing;
+
+import com.alrex.parcool.common.action.impl.Dodge.DodgeDirection;
+import com.github.exopandora.shouldersurfing.client.ShoulderSurfingImpl;
+import com.github.exopandora.shouldersurfing.config.Config;
+import com.github.exopandora.shouldersurfing.plugin.ShoulderSurfingRegistrar;
+import net.minecraft.client.Minecraft;
+import java.lang.reflect.Field;
+
+/**
+ * Compatibility class for the "Should Surfing" mod
+ */
+public class ShoulderSurfingCompat {
+    private static Minecraft mc = Minecraft.getInstance();
+    private static boolean isLoaded = false;
+    private static Object configClient = null;
+    static {
+        try {
+            // Try to load the Config class dynamically
+            Class<?> configClass = Class.forName("com.github.exopandora.shouldersurfing.config.Config");
+            Field clientField = configClass.getField("CLIENT");
+            configClient = clientField.get(null);
+            ShoulderSurfingRegistrar.getInstance().registerCameraCouplingCallback(new ShoulderSurfingDecoupledCamera());
+            isLoaded = true;
+        } catch (ClassNotFoundException | NoSuchFieldException | IllegalAccessException e) {
+            isLoaded = false;
+        }
+    }
+
+    public static Boolean isCameraDecoupled() {
+        if (!isLoaded) return false;
+        return ShoulderSurfingImpl.getInstance().isCameraDecoupled();
```

Comment:
Please use the api provided by `ShoulderSurfing.getInstance()`. 3rd party integrations should not directly use the implementation.